test/Microsoft.Health.Test.Common.Tests/Microsoft.Health.Test.Utilities.UnitTests.csproj

```diff
@@ -3,8 +3,6 @@
   <PropertyGroup>
     <TargetFrameworks>$(SupportedFrameworks);</TargetFrameworks>
     <NoWarn>$(NoWarn);NU1603</NoWarn>
-  </PropertyGroup>
-  <PropertyGroup>
     <DefineConstants>R5</DefineConstants>
```

Comment:
I guess while we're here, is there anything that depends on `<DefineConstants>R5</DefineConstants>`? That was a concept we are using in the FHIR repo

Reply:
I don't see it being used. I'll remove it